internal/service/project/resource_project.go

```diff
@@ -793,7 +793,7 @@ func UpdateProjectTeams(ctx context.Context, teamsAPI admin.TeamsApi, projectSta
 		_, err := teamsAPI.RemoveProjectTeam(ctx, projectID, teamID).Execute()
 		if err != nil {
 			apiError, ok := admin.AsError(err)
-			if ok && *apiError.ErrorCode != "USER_UNAUTHORIZED" {
+			if ok && apiError.ErrorCode != "USER_UNAUTHORIZED" {
```

Comment:
maybe not for the scope of this PR, it's easier to do:
```
if admin.IsErrorCode(err, "USER_UNAUTHORIZED") {
```

Reply:
nice, adjusted